### PR TITLE
tz (ticdc): fix timezone error

### LINKED ISSUE #3584: CDC:ErrLoadTimezone on RHEL 6.8
### What did you do?

Run `cdc server --pd=http://pd-host:2379`.

### What did you expect to see?

Everything to be normal.

### What did you see instead?

`[CDC:ErrLoadTimezone]readlink /etc/localtime: invalid argument`

### Versions of the cluster

TiCDC version (execute `cdc version`):

```console
master
```

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve? 
close #3584 

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
image
There are some error log, but cdc server run successfully.
Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### Release note 

```release-note
Fix timezone related error that cause cdc server can't run in some RHEL release version (6.8, 6.9 etc).
```

### CODE REVIEW

**Review by overvenus — APPROVED** [review]
LGTM

File: pkg/util/tz.go [hunk 1]
```diff
@@ -57,9 +57,6 @@ func GetLocalTimezone() (*time.Location, error) {
 	if time.Local.String() != "Local" {
 		return time.Local, nil
 	}
-	str, err := os.Readlink("/etc/localtime")
-	if err != nil {
-		return nil, cerror.WrapError(cerror.ErrLoadTimezone, err)
-	}
+	str := timeutil.InferSystemTZ()
 	return getTimezoneFromZonefile(str)
```
Comment (amyangfei) [inline pkg/util/tz.go:61]: Seems `getTimezoneFromZonefile` can be simplified after we use infer system tz.

**Review by amyangfei — APPROVED** [review]

File: pkg/util/tz.go (hunk 1 above)
Comment (asddongmen) [reply pkg/util/tz.go:61]: ok

**Pull request merged**